main.go

```diff
@@ -99,15 +109,31 @@ func main() {
 			Usage: "Configure private registry TLS paths, syntax should be <registry url>,<ca cert path>,<cert path>,<key path>",
 			Value: &config.RegistriesTLS,
 		},
+		&cli.StringFlag{
+			Name:        "log-file",
+			Usage:       "Path to log file on disk",
+			Destination: &logFile,
+			Value:       "/var/lib/rancher/migration-agent/agent.log",
+		},
 	}
 	app.Action = run
-
 	if err := app.Run(os.Args); err != nil {
 		logrus.Fatal(err)
 	}
 }
 
 func run(c *cli.Context) {
+	// set up logging to disk
+	if err := os.MkdirAll(filepath.Dir(logFile), 0755); err != nil {
+		logrus.Fatalf("failed to create logging directory: %v", err)
+	}
+	f, err := os.OpenFile(logFile, os.O_WRONLY|os.O_CREATE, 0755)
```

Comment:
Anyone can read this file? Migration logs tend to expose unexpected things over time, maybe we should lock this down more?

Reply:
will fix